polyfills/smoothscroll/detect.js

```diff
@@ -1,26 +1 @@
-('document' in self && 'documentElement' in self.document && 'style' in self.document.documentElement && 'scrollBehavior' in document.documentElement.style) || (function () {
-    try {
-        var supportsSmoothScroll = false;
-
-        var scrollOptions = {
-            top: 0,
-            left: 0
-        };
-
-        Object.defineProperty(scrollOptions, 'behavior', {
-            get: function () {
-                supportsSmoothScroll = true;
-                return 'smooth';
-            },
-
-            // Ensure this property lasts through cloning / destructuring:
-            enumerable: true
-        });
-
-        document.body.scrollTo(scrollOptions);
-
-        return supportsSmoothScroll;
-    } catch (e) {
-        return false;
-    }
-})();
\ No newline at end of file
+'document' in self && 'documentElement' in self.document && 'style' in self.document.documentElement && 'scrollBehavior' in document.documentElement.style;
```

Comment:
Suggestion from @romainmenke -- https://github.com/Financial-Times/polyfill-library/pull/903#issuecomment-716687789

```suggestion
('document' in self && 'documentElement' in self.document && 'style' in self.document.documentElement && 'scrollBehavior' in document.documentElement.style) || (function () {
	try {
		var supportsSmoothScroll = false;
		
		var scrollOptions = {
			top: 1,
			left: 0
		};
		
		Object.defineProperty(scrollOptions, 'behavior', {
			get: function () {
				supportsSmoothScroll = true;
				return 'smooth';
			},
			
			// Ensure this property lasts through cloning / destructuring:
			enumerable: true
		});
		
		var a = document.createElement('DIV');
		var b = document.createElement('DIV');
		a.setAttribute('style', 'height: 1px; overflow: scroll;');
		b.setAttribute('style', 'height: 2px; overflow: scroll;');
		
		a.appendChild(b);
		a.scrollTo(scrollOptions);
		
		return supportsSmoothScroll;
	} catch (e) {
		return false;
	}
})();
```